libs/testing/e2e/README.md

```diff
@@ -58,12 +58,10 @@ yarn e2e <app-name>
    yarn e2e <app-name> --skip-nx-cache
   ```
 
-- **Run a Specific Project**: Run only the tests defined under a specific project in your Playwright config:
+- **Run with a Specific Tag**: Run only the tests tagged with a specific tag:
 
   ```bash
-  yarn e2e <app-name> -- --project=smoke
-  yarn e2e <app-name> -- --project=acceptance
-  yarn e2e <app-name> -- --project=everything
+   yarn e2e <app-name> --grep @fast
```

Comment:
Add more examples? How would I run multiple tags? Or _not_ something? At least link to [docs](https://playwright.dev/docs/test-annotations#tag-tests).